.env.template

```diff
@@ -68,5 +68,9 @@ PFS_SERVICE_FEE=100
 # PFS_OPERATOR=<OPERATOR_NAME>
 # PFS_INFO_MESSAGE=<PFS_INFO_MESSAGE>
 
+## Settings for COMPRESS_STATE utility (optional)
+## rooms to compress: list of fully qualified room names, space separated
+# COMPRESS_ROOMS="#room1:homeserver #room2:homeserver #room3:homeserver"
```

Comment:
What happens when this is not set? We should either mention the default behavior or require this to be set to make it obvious.